etc/config/csharp.defaults.properties

```diff
@@ -0,0 +1,20 @@
+# Default settings for C++
+compilers=dotnetcore21_64
+defaultCompiler=dotnetcore21_64
+postProcess=
+# demangler=c++filt
+# demanglerClassFile=./demangler-cpp
+objdumper=objdump
+#androidNdk=/opt/google/android-ndk-r9c
+options=
+supportsBinary=true
+binaryHideFuncRe=^(__.*|_(init|start|fini)|(de)?register_tm_clones|call_gmon_start|frame_dummy|\.plt.*)$
```

Comment:
This seems like a copy-pasted C++ specific?

Reply:
Removed the copy-pasta, only necessary properties remain.